examples/create-react-app/BUILD.bazel

```diff
@@ -54,6 +55,9 @@ react_scripts(
     data = _RUNTIME_DEPS + [
         "@npm//@types",
     ],
+    env = {
+        "BUILD_PATH": "./build",
```

Comment:
oh, super interesting. this could be used to avoid the chdir altogether I imagine, by running with the default bazel working directory and telling CRA to output to the `$(@D)` bazel-out directory. Maybe worth trying as a follow-up PR?